src/style.css

```diff
@@ -2,2 +1,70 @@
+.first {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
+
+.navigation-item {
+  display: inline-block;
+  margin-left: 20px;
+}
+
+.navigation-item:first-child {
+  margin-left: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+}
+
+.navigation-link {
+  text-decoration: none;
+  color: black;
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-transform: uppercase;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
There should be a fallback font in case 'Roboto' is not available. This will ensure that your website still looks presentable even if the primary font fails to load. You can use a generic font family such as 'sans-serif' as a fallback.